test/support/coverage.rb

```diff
@@ -0,0 +1,43 @@
+module Coverage
```

Comment:
I don't mind changing it either way :) Just let me know. Thanks for taking the time to review this!